packages/stratimikos/test/stratimikos_jacobi_tpetra.xml

```diff
@@ -0,0 +1,84 @@
+<ParameterList>
```

Comment:
Where is the actual test definition? And shouldn't the xml be copied to the build dir?

Reply:
Thank you, @cgcgcg. I added those to ctest.